library/std/src/env.rs

```diff
@@ -92,7 +92,9 @@ pub struct VarsOs {
 ///
 /// The returned iterator contains a snapshot of the process's environment
 /// variables at the time of this invocation. Modifications to environment
-/// variables afterwards will not be reflected in the returned iterator.
+/// variable values afterwards will not be reflected in the returned iterator
+/// but modifications to the environment using [`set_var`] and [`remove_var`]
+/// will appear.
```

Comment:
This change suggests that using `set_var/remove_var` will change which items an already created iterator will return.